Fix weapon type detection

baharclerode opened the pull request:
Due to a capitalization issue, all weapon rolls from the character sheets are considered `ranged`; This causes `meleeDamageMod` to not be added to melee damage rolls, and probably a host of other quirky issues as well.

DerDiemen commented:
Hey there!

Thank you for your support, that‘s a good catch! :) Unfotunately I may just have revisited and refactored the whole weapon detection part and streamlined it ovrr all sheets which may have already corrected this particular issue. (I‘m off this weekend but am happy to let you know later)

In the meantime please do not send pull requests directly to the master as this is a live testing branch for some people (including myself). Always use „Development“ since that‘s the branch to break stuff without regrets :)

All the best

baharclerode commented:
> Always use „Development“

Easy enough! I didn't see any guidelines around contributions, so I just used the default branch. I've switched the PR to `Development`.

> I may just have revisited and refactored the whole weapon detection part

Not a problem, such is the nature of distributed software development :) If you're planning to release that in the next week or two, then no need to make a bugifx just for this, but if it's going to be a while before the refactor is ready to release, a small bugfix would certainly be appreciated! I'm not familiar with the release process for VTT mods, so I don't have any context for how much work is involved.

I really appreciate the work you've done to make this mod! My group recently switched to Eclipse Phase and this has made the jump so much easier than it otherwise would have been.

DerDiemen commented:
Hey @baharclerode !
So glad to hear that you and your people are enjoying the system :) Really means a lot to me!

I will have a look into this fix as soon as I am back home today/tomorrow and will merge it or let you know whether I have already fixed it. 

In the meantime I thought the exact same thing “how are they supposed to know how to contribute, you should include it in the readme!” ^^ - Anywho: If you want to contribute in a planned way during upcoming releases I’m happy to invite you to the dev server (no secret that it’s basically me talking to myself there most of the time, but at least we would have a place to discuss features and stuff). That way it’s easier to distribute work if you like :) 

Let me know if you’re interested by just pinging “laphaela” and we’ll sit together. (There’s also a fairly filled kanban board to get a feeling for further planned updates.)

DerDiemen commented:
Has been fixed during weapon-refactoring.
